packages/components/src/button/style.scss

```diff
@@ -119,13 +119,8 @@
 			background-size: 100px 100%;
 			// Disable reason: This function call looks nicer when each argument is on its own line.
 			/* stylelint-disable */
-			background-image: linear-gradient(
-				-45deg,
-				theme(button) 28%,
-				color(theme(button) shade(20%)) 28%,
-				color(theme(button) shade(20%)) 72%,
-				theme(button) 72%
-			);
+			background-image:
+ linear-gradient(-45deg, theme(button) 28%, color(theme(button) shade(20%)) 28%, color(theme(button) shade(20%)) 72%, theme(button) 72%);
```

Comment:
Any reason for this change other than personal preference?

Gradients always give me headaches, and although it's not really a coding pattern to split on to multiple lines, when actually working on these gradients and tweaking them, it really helps that they're on multiple lines. I'd prefer the old version unless there's a good reason to change this.

Reply:
Oooh interesting. Stylelint changed that automatically so I guess the `/* stylelint-disable */` comment above is being ignored for some reason. I'll revert that as it's unrelated to this changset 😛

Reply:
Done, thanks for the review @jasmussen !